libs/shared/contentful/src/lib/contentful.manager.spec.ts

```diff
@@ -77,6 +82,33 @@ describe('ContentfulManager', () => {
     );
   });
 
+  describe('getEligibilityContentByOffering', () => {
+    it('should return empty result', async () => {
+      const queryData = EligibilityContentByOfferingQueryFactory({
+        offeringCollection: { items: [] },
+      });
+      mockClient.query = jest.fn().mockReturnValue(queryData);
+      const result = await manager.getEligibilityContentByOffering('test');
+      expect(result).toBeInstanceOf(EligibilityContentByOfferingResultUtil);
+    });
+
+    it('should return successfully with results', async () => {
+      const apiIdentifier = 'test';
+      const offeringResult = [
+        EligibilityContentOfferingResultFactory({ apiIdentifier }),
+      ];
+      const queryData = EligibilityContentByOfferingQueryFactory({
+        offeringCollection: {
+          items: offeringResult,
+        },
+      });
+      mockClient.query = jest.fn().mockResolvedValue(queryData);
+      const result = await manager.getEligibilityContentByOffering('test');
```

Comment:
Should this use the const from L96?
```suggestion
      const result = await manager.getEligibilityContentByOffering(apiIdentifier);
```